content/en/platform/corda/5.2/deploying-operating/deployment/upgrading521/_index.md

```diff
@@ -0,0 +1,517 @@
+    weight: 5000
+---
+# Upgrading from 5.2 to 5.2.1
+
+This section describes how to upgrade a Corda cluster from version 5.2 to 5.2.1.
+
+{{< note >}}
+You cannot upgrade Corda 5.1 to 5.2.1. You must upgrade Corda to version 5.2 first. For information on how to do it, see [Upgrading from 5.1 to 5.2]({{< relref "../upgrading/_index.md" >}}).
+{{< /note >}}
+
+The following section includes script excerpts which you can use as a basis for performing upgrades. The entire script is included at the end. The script is based on upgrading a cluster running on Docker Desktop on a local host using the R3-provided Corda Helm chart for installing PostgreSQL and Kafka. However, you must tailor this script for the specific environment being updated.
+
+The examples provided in this section assume that you installed Corda 5.1 in a namespace called `corda`. This is different to other deployments.
+
+To perform an upgrade, you must fulfill the required [prerequisites](#prerequisites) and go through the following steps:
+
+1. [Back Up the Corda Database](#back-up-the-corda-database)
+2. [Test the Migration](#test-the-migration)
+3. [Migrate Data From Kafka Topics to the Cluster Database](#migrate-data-from-kafka-topics-to-the-cluster-database)
+4. [Scale Down the Running Corda Worker Instances](#scale-down-the-running-corda-worker-instances)
+5. [Migrate the Corda Cluster Database](#migrate-the-corda-cluster-database)
+6. [Optional: Clear Down Key Rotation Stale Data](#optional-clear-down-key-rotation-stale-data)
+7. [Update Kafka Topics](#update-kafka-topics)
+8. [Launch the Corda 5.2.1 Workers](#launch-the-corda-521-workers)
+
+For information about how to roll back an upgrade, see [Rolling Back]({{< relref "rolling-back521.md" >}}).
+
+Following a platform upgrade, Network Operators should upgrade their networks. For more information, see [Upgrading an Application Network]({{< relref "../../../application-networks/upgrading/_index.md" >}}).
+
+## Prerequisites
+
+Corda 5 relies on certain underlying prerequisites, namely Kafka and PostgreSQL, administered by the Cluster Administrators.
+
+{{< note >}}
+This guide assumes that the Cluster Administrator assigned to upgrade Corda has full administrator access to these prerequisites.
+{{< /note >}}
+
+Developers, including customer CorDapp developers, or those trialing Corda, can use the R3-provided [Corda Helm chart]({{< relref "../deploying/_index.md#download-the-corda-helm-chart" >}}) which installs these prerequisites. The Corda Helm chart can also configure Corda so it can reach these prerequisites, allowing a quick and convenient installation of Corda 5.
+
+Customers in production are not expected to follow this path, and generally use managed services for these prerequisites. They also impose significant privilege restrictions in terms of who can administer these services. This guide cannot provide instructions on how to gain administrator access to customer-managed or self-hosted services.
```

Comment:
Calling customers "they" in a customer facing document might seem a little abrupt. Perhaps `There are likely to be significant privilege restrictions in terms of who can administer these services.`